src/braket/jobs/quantum_job_creation.py

```diff
@@ -139,6 +139,9 @@ def prepare_quantum_job(
     _validate_params(param_datatype_map)
     aws_session = aws_session or AwsSession()
     device_config = DeviceConfig(device)
+
+    if instance_config and instance_config.instanceCount > 1 and not device.startswith("local:"):
```

Comment:
Do we need an upper limit?

Reply:
I agree. Let's move to the service and simply return the API validation results.